x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/comparison/Comparisons.java

```diff
@@ -25,6 +25,10 @@ public static Boolean eq(Object l, Object r) {
         return i == null ? null : i.intValue() == 0;
     }
 
+    public static Boolean eqIgnoreCase(Object l, Object r) {
```

Comment:
Who's using this method?
If needed see StringComparisons inside EQL

Reply:
Removed (together with all the other changes in QL)